mesop/components/input/input.ts

```diff
@@ -35,6 +36,14 @@ export class InputComponent {
         debounceTime(150),
       )
       .subscribe((event) => this.onInputDebounced(event));
+    // See keyDown event for explanation for why we have a special case for Safari.
+    if (
+      navigator.userAgent.indexOf('Safari') > -1 &&
```

Comment:
minor: you can use String.prototype.includes for a bit more straightforward check: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes